src/solver/simulation/sim_calcul_adequation.cpp

```diff
@@ -274,11 +274,11 @@ void SIM_RenseignementValeursPourTouteLAnnee(const Antares::Data::Study& study,
     }
 }
 
-void SIM_RenseignementProblemeHoraireAdequation(long Heure)
+void SIM_RenseignementProblemeHoraireAdequation(long Heure, uint numSpace)
```

Comment:
**numSpace** is not a great name, and I guess it is not what you think it is : it's the number of the year in the set of years run in parallel. It is not an absolute number in the range of years of the simulation.
I guess what you're after is more :
`state[numSpace].year`
which is updated in solver.hxx (184)
So, when you use it to get a NTC ts number, use the previous instead

Reply:
ok

Reply:
my bad, **NumeroChroniquesTireesParInterconnexion** is updated before running each set of parallel years and its dimension is the number of years in a set of parallel years

Reply:
I'll keep it that way for Adequacy/Economy, and remove the unnecessary argument for Adequacy Draft, since `numSpace` is always 0 in this mode.